www/src/pages/components/avatar/usage/index.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Avatar
```

Comment:
@danoc This header doesn't render locally for some reason. 

🤔 Does it need to be defined after the imports or somehow differently?

Reply:
It seems to be working in the deploy preview of this PR:

https://deploy-preview-703--thumbprint.netlify.app/

But I think I know what the issue is. I've created this to track it:
https://github.com/thumbtack/thumbprint/issues/706